story/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.utils import timezone
+from house.models import House
+
+
+class Story(models.Model):
+    house = models.ForeignKey(House, on_delete=models.CASCADE)
+    story_date = models.DateTimeField(default=timezone.now)
+    story_text = models.TextField()
+
```

Comment:
I would change the variable name to align with the ERD you have made. 
for example, date is supposed to be the date the story was last edited. 

what about CRUD functions? for example: https://github.com/redhat-beyond/Xpense/blob/e00f90465ec146a84a3acd052120c644001f83b6/house/models.py#L60